pages/common/bfg.md

```diff
@@ -0,0 +1,16 @@
+# bfg
+
+> Remove large files or passwords from Git history like git-filter-branch
+> More information: <https://rtyley.github.io/bfg-repo-cleaner/>.
+
+- Remove a file with sensitive data and leave the latest commit untouched:
+
+`bfg --delete-files {{FILE-WITH-SENSITIVE-DATA}}`
+
+- Replace all text listed in passwords.txt wherever it can be found in the repository's history:
+
+`bfg --replace-text {{passwords.txt}}`
+
+- After the sensitive data is removed, you must force push your changes to GitHub:
+
+`git push --force`
```

Comment:
We normally don't reference other commands. We may add something to one of the other command, like these changes are only commited, not pushed to origin

Reply:
@bl-ue what's your opinion on this

Reply:
Also this needs to not be Github, just origin since gitlab and bitbucket do exist

Reply:
There's a few problems with this example:
1. As @CleanMachine1 points out, it's a different command other than the one that we're documenting, but that's of the least concern
2. The command assumes that the repository is connected to a remote, which it may, in fact, not be
3. GitHub isn't the only remote repository service ;)

I suggest moving this information to a new line in the page description, something like
```
> Note: you will need to force-push if your repository is connected to a remote.
```

Reply:
Thank you so much for your detailed feedback! I'll make these changes within a day or two when I get some time 😊